process/block/baseProcess.go

```diff
@@ -1123,17 +1123,21 @@ func (bp *baseProcessor) RevertAccountState(_ data.HeaderHandler) {
 
 func (bp *baseProcessor) commitAll(headerHandler data.HeaderHandler) error {
 	if headerHandler.IsStartOfEpochBlock() {
-		currentEpoch := headerHandler.GetEpoch()
-		lastEpoch := currentEpoch - 1
-		if lastEpoch < 0 {
-			lastEpoch = 0
-		}
-		return bp.commitInEpoch(currentEpoch, lastEpoch)
+		return bp.commitInLastEpoch(headerHandler.GetEpoch())
 	}
 
 	return bp.commit()
 }
 
+func (bp *baseProcessor) commitInLastEpoch(currentEpoch uint32) error {
```

Comment:
Rename to commitInPreviousEpoch?
"Last epoch" to me is similar to current epoch